openedx/core/djangoapps/catalog/utils.py

```diff
@@ -101,18 +107,77 @@ def get_program_types(user=None):  # pylint: disable=redefined-builtin
         return []
 
 
-def get_programs_data(user=None):
-    """Return the list of Programs after adding the ProgramType Logo Image"""
+def _get_program_instructors(program):
```

Comment:
Currently for courses that run on WL sites, instructors are configured via the Schedule & Details page in Studio and persisted on the course in Mongo. This instructor data is not being pulled by the catalog service's refresh_course_metadata job. We thought about updating refresh_course_metadata to pull this data into the catalog, but I think it makes more sense to wait until publisher fully supports configuring a course (including instructor information). Once we have a single tool for configuring course metadata, we can change this code to pull instructor data directly from the catalog service.